custom-targets/util/clouddeploy/clouddeploy.go

```diff
@@ -29,9 +29,11 @@ import (
 	"strings"
 
 	"cloud.google.com/go/storage"
-	"github.com/mholt/archiver/v3"
 )
 
+// GitCommit SHA to be set during build time of the binary.
+var GitCommit = "unknown"
```

Comment:
How exactly is this populated?

Reply:
When compiling the custom target binary in the Dockerfile, the go build steps gets `-ldflags=` parameter that can be used to set the value of this var.